### ISSUE TITLE
Saving "scenes" or "locations"

### ISSUE THREAD

**Issue by fijal** [lightscapevr/vr-sketch-4#29]
A lot of people want to be able to save "scenes" or "locations" in the model. I'm not sure exactly what sketchup scenes do (I *think* they also have a cuts and some parts hidden), but maybe reusing them would be useful.

Having an extra menu where the "scenes" are saved for quick navigation is probably super useful.

**Comment by arigo** [comment]
Ah, and then we can add the location "start-up view in the warehouse" to that list, and thus make a clearer way to teleport back to the start-up view.

**Comment by fijal** [comment]
I think there are two elements of UI here:

1. Have a "scene selector" in sketchup, from sketchup scenes

2. Have a "scene selector" in one way or another in VR. Maybe something like material selection would work?

**Comment by arigo** [comment]
Or just  take all "scenes" from SketchUp, and display them in the scene selector in VR.  Is there a reason for why we should choose in SketchUp which scenes we want?

**Comment by fijal** [comment]
No, the idea would be to have "teleport to scene" *also* in sketchup, so you don't have to teach someone how to navigate UI to be able to do this

**Comment by arigo** [comment]
Sorry, still unsure what you're saying.  I understand your comments as saying "we would need to add a new UI on the 2D monitor".  But that seems at odd with "don't have to teach someone how to navigate UI".  What I am saying is that SketchUp has its own way to handle scene views and moving between them.  What we need, as a first step, is just to synchronize with this list of scene views so that, in VR, we can pick one of them and be teleported to the same location.  Right?

**Comment by arigo** [comment]
(Tentatively resolved, writing it down here so that I remember: the "teach the UI" in fijal's comment doesn't mean "adding a 2D UI is a problem" but "it should work even if the person in VR is not taught how to navigate the VR UI")
